site/specs/numbers.json

```
@@ -6320,140 +6320,133 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieves a list of in-service phone numbers associated with the account ID. There are multiple search parameters for searching for in-service numbers:</p>\n\n<ul>\n    <li>size and page for pagination</li>\n    <li>area code</li>\n    <li>Npa-Nxx</li>\n    <li>LATA</li>\n    <li>state</li>\n    <li>rate center</li>\n</ul>\n",
-        "operationId": "GET /accounts/{accountId}/inserviceNumbers",
+        "description": "Retrieves a list of in-service phone numbers associated with the account ID",
+        "summary": "Fetch in-service phone numbers.",
```

```suggestion
        "summary": "Fetch in-service phone numbers",
```

The summary is what the operation will be named on the UI, I don't think it should have a period in it.

---

fixed